pahole/pkg.yaml

```diff
@@ -0,0 +1,29 @@
+name: pahole
+dependencies:
+  - stage: base
+  - stage: elfutils
+  - stage: cmake
+steps:
+  - sources:
+      - url: http://deb.debian.org/debian/pool/main/d/dwarves/dwarves_1.23.orig.tar.xz
```

Comment:
probably use this: https://git.kernel.org/pub/scm/devel/pahole/pahole.git/snapshot/pahole-1.23.tar.gz